### ISSUE TITLE
daterangepicker: Hour and minute dropdown not working in firefox and IE11

### ISSUE THREAD

**Issue by hrichakabir** [dangrossman/bootstrap-daterangepicker#1311]
@dangrossman : I am using bootstrap-daterangepicker 2.1.19, in chrome it works perfectly. But in firefox and IE 11 not able to select or change hour and minute as hour and minute options are not displaying.  
Firefox version : 50.0

**Comment by dangrossman** [comment]
I just checked both Firefox 50 and IE 11, and the hour and minute dropdowns were visible and usable at daterangepicker.com

**Comment by berarma** [comment]
Hi @hrichakabir. I've been fighting this problem and I've found a solution for my case. I had the daterangepicker inside a bootstrap modal and the dropdown for hours and minutes wouldn't show up. The modal is inside a form. It did a very small flash like appearing very briefly and then disappearing very fast, almost imperceptible.

I'm sharing some of the things that I tried and also the solution that worked for me.

Try running Firefox in safe mode to discard some bug related to extensions or themes. I think safe mode also disables hardware acceleration. https://support.mozilla.org/es/questions/1150392

Try disabling hardware acceleration in advanced settings. https://support.mozilla.org/es/questions/1075185

In my case, I've found out that it wouldn't work when the modal dialog was defined like this:
```
<div class="modal fade" tabindex="-1" role="dialog">
```
But it would work when I removed the ```tabindex``` attribute:
```
<div class="modal fade" role="dialog">
```

This is with Firefox 52.1.0 (64bit) running on Debian GNU/Linux 8.
